qt/widgets/common/src/AlgorithmHistoryWindow.cpp

```diff
@@ -509,6 +517,23 @@ void AlgorithmHistoryWindow::doUnroll(const std::vector<int> &unrollIndicies) {
 
 void AlgorithmHistoryWindow::doRoll(int index) { m_view->roll(index); }
 
+void AlgorithmHistoryWindow::unrollAll(int state) {
+  // Iterate all items in tree which have children algorithms to be unrolled
+  QTreeWidgetItemIterator it(m_Historytree,
+                             QTreeWidgetItemIterator::HasChildren);
+
+  auto head = m_Historytree->headerItem();
+  std::cout << head->text(0).toStdString() << std::endl;
```

Comment:
Could we remove the `cout`? I assume this was intended for debugging?

Reply:
Good spot, I've removed this now. You are right this was for debugging/clarification